IronGram/src/main/java/com/theironyard/services/ExecutionerOfPhotos.java

```diff
@@ -0,0 +1,28 @@
+package com.theironyard.services;
+
+import com.theironyard.entities.Photo;
+
+import java.util.List;
+
+/**
+ * Created by EddyJ on 7/31/16.
+ */
+public class ExecutionerOfPhotos implements Runnable {
```

Comment:
This is a little weird of a name.  Technically this isn't an Executioner of anything.  Really a name like PhotoRunner or PhotoRemover might be more apt given Executor is a thing in java.